torntools/scripts/content/faction/ttFaction.js

```diff
@@ -417,15 +417,18 @@ function showNNB() {
 }
 
 function fullInfoBox(page) {
-	let info_box;
+	let info_box, facDescription;
 	if (getSearchParameters().get("step") === "profile") {
 		info_box = doc.find("#factions div[data-title='description']").nextElementSibling;
 	} else if (page === "main") {
 		info_box = doc.find("div[data-title='announcement']").nextElementSibling;
+		facDescription = info_box;
 	} else if (page === "info") {
 		info_box = doc.find("#faction-info .faction-info-wrap.faction-description .faction-info");
 	}
 
+	facDescription ? facDescription = info_box : facDescription = info_box.parentElement.find("div.faction-description");
```

Comment:
It would be better to do `facDescription = x ? a : b` directly. It's shorter and more clear at first glance.